tests/torchtune/utils/test_collate.py

```diff
@@ -6,9 +6,13 @@
 
 # (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
 
+from unittest import mock
+
+import pytest
 import torch
+from torchtune.utils import torch_version_ge
 
-from torchtune.utils.collate import padded_collate
+from torchtune.utils.collate import padded_collate, padded_collate_packed
 
 
 class TestBatchPadSequence:
```

Comment:
given how these tests were modified, i am understanding that we you were planning to support efficient attention anymore. Maybe thats ok, but it implies that user must have:

torch>=2.5
compile will always be on

In principle, that works for me, but maybe its worth some discussion? Sorry if it was already